types/priv_validator.go

```diff
@@ -105,13 +97,14 @@ func LoadPrivValidator(filePath string) *PrivValidator {
 	if err != nil {
 		Exit(err.Error())
 	}
-	privVal := wire.ReadJSON(&PrivValidator{}, privValJSONBytes, &err).(*PrivValidator)
+	privVal := PrivValidator{}
```

Comment:
`&PrivValidator{}` and `new(PrivValidator)` are the same.  We could take the pointer here.

In all the go code I read before coming to tendermint (which was various repos, but not extensive), I saw the first idiom much more often and it is easier for me to parse visually.  I think you like the `new` format.  Is there a reason, or just style?

We should add this to coding guidelines.  But I wonder why you like new? (except for instantiating pointers to interfaces which no sane code should do)